core/storage/user/gae_models.py

```diff
@@ -2023,12 +2133,17 @@ def get_deletion_policy():
         """
         return base_models.DELETION_POLICY.DELETE
 
+    @staticmethod
+    def get_export_method():
+        """Model is exported as multiple unshared instances."""
+        return base_models.EXPORT_METHOD.MULTIPLE_UNSHARED_INSTANCES
+
     @classmethod
     def get_export_policy(cls):
         """Model contains user data."""
         return dict(super(cls, cls).get_export_policy(), **{
             'user_id': base_models.EXPORT_POLICY.NOT_APPLICABLE,
-            'score_category': base_models.EXPORT_POLICY.EXPORTED,
+            'score_category': base_models.EXPORT_POLICY.NOT_APPLICABLE,
```

Comment:
Why this change?

Reply:
see above (was caught as not exported in the export_data).